README.md

```diff
@@ -8,6 +8,15 @@ ES6 and Webflow.
 
     yarn run import-webflow zipfile.zip
 
+
+## Deployment 
+
+All updates to `master` will automatically deploy to Netlify.
```

Comment:
This is production correct? If so, then I think we should be even more explicit.

```suggestion
All updates to `master` will automatically deploy to the production instance on Netlify.
```